tests/jobserver/test_views.py

```diff
@@ -523,6 +530,192 @@ def test_jobrequestzombify_unknown_jobrequest(rf):
         JobRequestZombify.as_view()(request, pk="99")
 
 
+@pytest.mark.django_db
+def test_orgcreate_get_success(rf):
+    oxford = OrgFactory(name="University of Oxford")
+    datalab = OrgFactory(name="DataLab")
+
+    request = rf.get(MEANINGLESS_URL)
+    request.user = UserFactory(is_superuser=True)
+    response = OrgCreate.as_view()(request)
```

Comment:
I'm sure we've been here before and you've explained this before and I'm sure it makes sense really, but I'm expecting this test to test the create *view* _creates something_, but actually we're creating something and then testing that the view does the right kind of thing assuming it's successfully created objects - right?

 I wonder if there's some more expressive way of making it clear what's going on?  This confuses me every time...!

Reply:
No, this is testing the `GET` handler of `OrgCreate`, whose template lists existing Orgs as well as the create form.

So the test is creating some objects, calling OrgCreate with a GET request, checking the orgs we constructed beforehand are in the response's template context.

Reply:
I see. I wish there were a clearer way to express this but I can see it's all in there on reflection, so no clever ideas I'm afraid